### PR TITLE
2022-09-20-Wintermute

### PR DESCRIPTION
**Pull request by JediFaust**

### CODE REVIEW

**Comment by jhirschkorn** [comment]
please add a citation/reference for the following bot comment(s):

> The attack was executed through a brute force private key compromise. ❌
> There is no evidence in the search results to suggest that the attack on Wintermute was executed through a brute force private key compromise. The articles only mention that the company was hacked and lost $160 million worth of cryptocurrency.

File: content/attacks/posts/2022-09-20-Wintermute.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+date: 2022-09-20
+custodians: Wintermute
+categories: Custodian, Trading Platform
+title: "Wintermute Incurs $160 Million Loss from Brute Force Private Key Compromise Linked to Profanity's Vulnerability"
+---
+
+## Summary
+
+On September 20, 2022, Wintermute, a London-based algorithmic market maker offering liquidity across Centralized Finance (CeFi) and Decentralized Finance (DeFi) exchanges and over-the-counter (OTC) deals, was the victim of a security breach. The exploit resulted in a loss of approximately $160 million, impacting 90 different assets including stable coins, Bitcoin, Ether, and various altcoins. The attack was executed through a brute force private key compromise. The suspected vulnerability originated from Profanity, a service Wintermute used for generating vanity addresses, despite efforts to blacklist their Profanity-associated accounts after the vulnerability became known.
```
Comment (jhirschkorn) [inline content/attacks/posts/2022-09-20-Wintermute.md:10]: add a reference for brute force key compromise

File: content/attacks/posts/2022-09-20-Wintermute.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+## Attackers
+
+The identity of the attackers remains unknown. The Ethereum address linked to the attacker and currently holding all stolen funds:
```
Comment (jhirschkorn) [inline content/attacks/posts/2022-09-20-Wintermute.md:14]: add current as of date to address holding funds

File: content/attacks/posts/2022-09-20-Wintermute.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- **September 15, 2022 6:00 AM UTC:** 1inch Network [drew attention to the issue](https://twitter.com/1inch/status/1570291260002373633) in their blog.
+- **September 15, 2022 8:42 PM UTC:** [First malicious transaction](https://etherscan.io/tx/0xdf5d8d087813b2c0efed72cf3cee5b2d9beb16fca87ecfa7a78740260950fee8) of [$3.3 Million Profanity hack](https://insidebitcoins.com/news/hackers-drain-3-3m-from-profanity-wallets-despite-1inch-warning) was performed
+- **September 20, 2022 5:11 AM UTC:** [Malicious transaction](https://etherscan.io/tx/0xedd31e2a949b7957a786d44b071dbe1bc5abd5c57e269edb9ec2bf1af30e9ec4) affecting Wintermute's wallets was performed.
+- **September 20, 2022 8:03 AM UTC:** Wintermute's CEO, Evgeny Gaevoy, promptly [announces the theft](https://twitter.com/EvgenyGaevoy/status/1572134271011225601).
+
+## Security Failure Causes
+
+**Profanity's Vulnerability:** An inherent weakness in Profanity's code allowed the attacker to generate all potential keys for a vanity address by bruteforcing the private keys, scan associated accounts, and then steal the funds.
```
Comment (jhirschkorn) [inline content/attacks/posts/2022-09-20-Wintermute.md:34]: add citation if possible

**Review by jhirschkorn — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2022-09-20-Wintermute.md (hunk 3 above)
Comment (JediFaust) [reply content/attacks/posts/2022-09-20-Wintermute.md:34]: Done

File: content/attacks/posts/2022-09-20-Wintermute.md (hunk 2 above)
Comment (JediFaust) [reply content/attacks/posts/2022-09-20-Wintermute.md]: Done

File: content/attacks/posts/2022-09-20-Wintermute.md (hunk 1 above)
Comment (JediFaust) [reply content/attacks/posts/2022-09-20-Wintermute.md]: Done

**Review by jhirschkorn — APPROVED** [review]

**Pull request merged**